### ISSUE TITLE
Different branches, different dependencies

### ISSUE THREAD

**Issue by sarahmarshy** [stellar/new-docs#559]
I have needed to use your existing branches at the end of each tutorial step here - https://developers.stellar.org/docs/building-apps

One consistent thing that is wrong with each branch is the version of js-combinatorics 
`sep-24` branch uses an outdated version, causing a compile error - https://github.com/stellar/docs-wallet/blob/sep24/package.json#L38

Correct one is 1.4.5 per the `pay` branch - https://github.com/stellar/docs-wallet/blob/pay/package.json#L38

[This line](https://github.com/stellar/docs-wallet/blob/sep24/src/components/loader/loader.tsx#L36) in `sep-24` branch is invalid for js-combinatorics. It's [fixed](https://github.com/stellar/docs-wallet/blob/trustline/src/components/loader/loader.tsx#L27) in `pay` and `trustline`.

Basically, I expect these branched to be the "finished" app and should compile out of the box. Just a quick test of all of them could probably show you any errors I have missed.
